tests/network/detect_snicat_sni_exfiltration.test.yml

```diff
@@ -0,0 +1,10 @@
+name: Detect SNICat SNI Exfiltration Unit Test
+detections:
+- name: Detect SNICat SNI Exfiltration
+  file: network/detect_snicat_sni_exfiltration.yml
+  pass_condition: '| stats count | where count > 0'
+attack_data:
+- file_name: zeek-ssl.log
+  data: https://attack-range-attack-data.s3-us-west-2.amazonaws.com/T1041/zeek-ssl.log
```

Comment:
@davisshannon : Did you upload this log file to the s3 bucket. I am somehow not able to find it there?

Other than that, the story looks good! Thanks again for being a solid contributor